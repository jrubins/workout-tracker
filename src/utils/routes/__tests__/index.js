import { unStringifyQuery } from '../'

describe('routes utils', () => {
  describe('unStringifyQuery', () => {
    test('returns an empty object if there is no query string', () => {
      expect(unStringifyQuery(null)).toEqual({})
      expect(unStringifyQuery('?foo=bar&blah=ball')).toMatchSnapshot()
    })
  })
})
