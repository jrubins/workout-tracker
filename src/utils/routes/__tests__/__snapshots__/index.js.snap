// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`routes utils unStringifyQuery returns an empty object if there is no query string 1`] = `
Object {
  "blah": "ball",
  "foo": "bar",
}
`;
